### PR TITLE
BGDIINF_SB-1702: Show only one layer details menu  at a time

### PR DESCRIPTION
**Pull request by ltshb**
Now the layer details menu is only shown one at a time like on the old viewer.

Also updated the CONTRIBUTING.md from my understanding of it but not sure if it is correct. For this to work correctly I think it is also missing the initial version tag `v0.0.0` on master. @pakb if you agree I'll create this tag on master ?

[Test link](https://web-mapviewer.dev.bgdi.ch/feature_bgdiinf_sb-1702_layer_visibility/index.html)

### CODE REVIEW

File: tests/e2e/specs/layerDetails.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            .find('[class="menu-layer-list-item-details"]')
+            .as('wmsLayerDetail')
+            .should('be.visible')
+        // the other layer details should not be visible
+        cy.get('[data-cy="button-open-visible-layer-settings-test.wmts.layer"]')
+            .as('wmtsDetailBtn')
+            .parents('[class="menu-layer-list-item"]')
+            .find('[class="menu-layer-list-item-details"]')
+            .as('wmtsLayerDetail')
+            .should('not.be.visible')
```
Comment (pakb) [inline tests/e2e/specs/layerDetails.spec.js:25]: I'd prefer that we don't use CSS class selector, but instead that we add a custom HTML attribute `data-cy="..."` to any element we need to grab in our test (see [Cypress's best practices on that](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements))

**Review by pakb — CHANGES_REQUESTED** [review]
Great! Only one little thing in the test that I'd like to address.
Regarding your tagging of master, if that makes the GitHub action/plugin work, please do.

File: tests/e2e/specs/layerDetails.spec.js (hunk 1 above)
Comment (ltshb) [reply tests/e2e/specs/layerDetails.spec.js:25]: Good idea, thanks for the tip I'll change that.

**Comment by pakb** [comment]
Could you change the color of the activated cog button to red (using the class `text-danger` on it or something similar) ?
I think it's written somewhere in the ticket to do so 😉

**Comment by ltshb** [comment]
> Could you change the color of the activated cog button to red (using the class `text-danger` on it or something similar) ?
> I think it's written somewhere in the ticket to do so

Oops true I forget about that :+1:

**Review by pakb — APPROVED** [review]

**Pull request merged**